library/components/TTooltip.vue

```diff
@@ -93,29 +102,51 @@ export default {
       }
       this.hide();
     },
+    handleTooltipFocus(isFocused) {
+      this.isTooltipFocused = isFocused;
+      if (!isFocused) {
+        this.hide();
+      }
+    },
     show() {
       this.isVisible = true;
 
       this.$nextTick(() => {
         // Without $nextTick, behaviour is glitched.
         this.placeTooltip();
-      })
+      });
     },
-    hide() {
+    hide: window._.debounce(function () {
+      // Debounce will give user chance to hover to tooltip and keep it open, as long as cursor stays on it.
+      // This is useful when tooltip contains clickable links.
+      if (this.isTooltipFocused) {
+        // If user is interacting with tooltip
+        return;
+      }
       this.isVisible = false;
-    }
+    }, 100),
+  },
+  watch: {
+    // Watcher triggers placement logic as well.
+    isOpen(n) {
+      if (n) {
+        this.show();
+      } else {
+        this.hide();
+      }
+    },
```

Comment:
Change 6:
Without watcher there is no way to re-trigger the placement when open state is controlled by parent trigger.